BuildingSync.xsd

```diff
@@ -6091,6 +6091,18 @@
                       <xs:documentation>Type of certification held by an auditor team member.</xs:documentation>
                     </xs:annotation>
                   </xs:element>
+                  <xs:element name="AuditorYearsOfExperience" minOccurs="0">
+                    <xs:annotation>
+                      <xs:documentation>The number of years the energy auditor has been conducting audits professionally.</xs:documentation>
+                    </xs:annotation>
+                    <xs:complexType>
+                      <xs:simpleContent>
+                        <xs:extension base="xs:integer">
```

Comment:
Should we use `nonNegativeInteger` type here?
http://books.xmlschemata.org/relaxng/ch19-77247.html

Reply:
Is it something we can modify here?

Reply:
addressed f0cc13c